samples/notebookinstance_lifecycle_config/README.md

```diff
@@ -0,0 +1,35 @@
+# Notebook Instance Lifecycle Configuration sample
+
+## Prerequisites
+
+This sample assumes that you have completed the [common prerequisites](/samples/README.md).
+
+### Create a Notebook Instance Lifecycle Configuration
+This command creates a Notebook Instance Lifecycle configuration.
+Note: The onCreate and onStart fields must be in base64.
+
+In the example crd the value of the onCreate field decodes to: `echo "Entering onCreate"` and the value of the onStart field decodes to: `echo "Entering onStart"`
+```
```

Comment:
In all the other samples they contain a string saying something like `trainingjob.sagemaker.services.k8s.aws.amazon.com/my-training-job created` as an example https://github.com/aws-controllers-k8s/sagemaker-controller/tree/main/samples/training. Can you add that to stay consistent

Reply:
done